docker-compose.yml

```
@@ -88,6 +88,15 @@ services:
       - "6379:6379"
     restart: always
 
+  aksk:
+    image: sodafoundation/multi-cloud-aksk
+    environment:
+      - MICRO_REGISTRY=mdns
+      - IAM_HOST=IAM_HOST
```

populate during install

---

Raised PR : https://github.com/sodafoundation/installer/pull/465

---

You can use IAM_HOST as environment variable, so that no need to edit docker-compose.yml file for using it.
```suggestion
      - IAM_HOST=${IAM_HOST}
```